FogCreek/Glitch-Community
# Nova chance: Updated a bunch of packages and removed ProgressPromise

## Links
https://nova-chance.glitch.me

## GIF/Screenshots:
![image](https://user-images.githubusercontent.com/8932174/61404091-ad78f880-a8a4-11e9-8757-ff1ffb695a92.png)

## Changes:
- Updated a bunch of packages that will get extra use with ssr enabled (eslint and sentry both have major version updates that I didn't use because there are breaking changes) (removed @babel/polyfill which is deprecated in favor of core-js)
- Refactored the s3-updloader a bit so it doesn't modify the Promise prototype to send progress updates
- Fixed bugs with the progress bar component and with updating notifications

## How To Test:
Try uploading covers and avatars to the site, the progress bar should update properly
